plugins/CorePluginsAdmin/Commands/ListPlugins.php

```diff
@@ -56,10 +56,9 @@ protected function doExecute(): int
         if ($this->getInput()->getOption('json')) {
             $plugins = array_map(function ($plugin) {
                 $plugin["comment"] = !isset($plugin["activated"]) ? 'Plugin not found in filesystem.' : '';
-                if (isset($plugin["version"]) && !isset($plugin["activated"])) {
+                if (isset($plugin["version"]) && $plugin["activated"] === null) {
```

Comment:
That's basically just another way to perform the check, but makes no difference. As the code checks with `isset` in all other places as well, I'd revert that for consistency reasons.
```suggestion
                if (isset($plugin["version"]) && !isset($plugin["activated"])) {
```